src/client/components/ReviewPost/ReviewPost.styles.css

```diff
@@ -0,0 +1,35 @@
+.review-section {
+  max-width: 600px;
+  display: flex;
```

Comment:
Maybe grid works better?
When I started working on the component I made similar to this one, I also used flex but I think in the end grid made it easier. Up to you ofc.